dev.pulsar_edit.Pulsar.json

```diff
@@ -0,0 +1,88 @@
+{
+    "app-id": "dev.pulsar_edit.Pulsar",
+    "base": "org.electronjs.Electron2.BaseApp",
+    "base-version": "22.08",
+    "runtime": "org.freedesktop.Sdk",
+    "runtime-version": "22.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "/opt/Pulsar/pulsar",
+    "rename-desktop-file": "pulsar.desktop",
+    "rename-icon": "pulsar",
+    "separate-locales": false,
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--share=network",
+        "--device=dri",
+        "--filesystem=home",
+        "--talk-name=org.freedesktop.Notifications",
+        "--talk-name=org.freedesktop.secrets",
+        "--talk-name=org.gtk.vfs.*",
+        "--env=ELECTRON_TRASH=gio",
+        "--env=TMPDIR=/var/tmp"
+    ],
+    "modules": [
+        {
+            "sources": [
+                {
+                    "type": "file",
+                    "only-arches": ["x86_64"],
+                    "url": "https://github.com/pulsar-edit/pulsar/releases/download/v1.101.0-beta/Linux.pulsar_1.101.0-beta_amd64.deb",
+                    "sha256": "d73d3c4a43fa1c86690ad6b13861d0eaf7a436015e327b960e1d821453c3fa39"
+                },
+                {
+                    "type": "file",
+                    "only-arches": ["aarch64"],
+                    "url": "https://github.com/pulsar-edit/pulsar/releases/download/v1.101.0-beta/ARM.Linux.pulsar_1.101.0-beta_arm64.deb",
+                    "sha256": "4fac74aaf9109795c82a560996748542c9c427865f91aa6391f87ad05b150af7"
```

Comment:
Please configure x-data-checker as described at https://github.com/flathub/flatpak-external-data-checker/.

Reply:
How can I do this? Pulsar releases are prereleases so it won't show in https://api.github.com/repos/pulsar-edit/pulsar/releases/latest unlike how other projects are configured so I'm not sure how to do this (I've already tried other strategies but none seem to work).

Reply:
Well @barthalion since I can't seem to find the solution, I will just disable the external data checker if thats okay.